src/getAge.test.js

```diff
@@ -3,21 +3,29 @@ const { getAge } = require('./getAge')
 describe('getAge()', () => {
   it('throws an error when input is not a number', () => {
 
-  })
+    expect(getAge).toThrowError(new Error('Input should be number'))
+  });
 
   it('throws an error when input is greater than current year', () => {
+    const year = 2021;
+    //expect(getAge(year)).toThrowError(new Error('Input should be greater than current year'))
+    expect(year).toBeLessThanOrEqual(2022);
 
```

Comment:
In this expectation you are only testing your year variable that you defined in the test and not the getAge function. 

Your initial thought was more correct (the commented out code) but the year you are testing with is not. 
If you look at the implementation of getAge you will find that it errors if the given year is __greater__ than the current year, but you are testing with 2021. 
The reason it is not working is that if you use toThrowError this way, you must give a callback to expect. Please have a look at how we used toThrow in session and also have a look at the 
[documentation](https://jestjs.io/docs/expect#tothrowerror)

As you can see on the first test you also used toThrowError, but it is working fine. This is because you really gave a callback to expect not the result of the function call. (there are no parenthesis after getAge in first test)